flexmeasures/data/services/forecasting.py

```diff
@@ -30,7 +30,7 @@
 """
 The life cycle of a forecasting job:
 1. A forecasting job is born in create_forecasting_jobs.
-2. It is run in make_forecasts which writes results to the db.
+2. It is run in make_rolling_viewpoint_forecasts or make_fixed_viewpoint_forecasts, which write results to the db.
```

Comment:
`make_fixed_viewpoint_forecasts` does not exist yet, right? So I'd make this a TODO somewhere, rather than mentioning it like this.